src/layout/strategy/noop.js

```diff
@@ -0,0 +1,29 @@
+import d3 from 'd3';
+
+export default function() {
+
+    var containerWidth = 1,
```

Comment:
I'm not so sure about supplying scales to strategies. I think of them as more akin to our indicator calculators. They operate on the screen coordinate system.